File: dotnet/console/Console/Scenarios/NotifyFlowParticipantsScenario.cs
using System.Threading.Tasks;

namespace Console.Scenarios
{
    public class NotifyFlowParticipantsScenario : Scenario
    {
        /**
         * This scenario demonstrates how to notify participants 
         * of the flow.
         */
        public override async Task RunAsync()
        {
            // 1. Get a document Id
            var result = await createDocumentAsync();

            // 2. Get the document details
            var details = await SignerClient.GetDocumentDetailsAsync(result.DocumentId);

            // 3. Notify each participant individually if necessary
            //    Note: Only participants with pending actions are notified.
            foreach (var flowAction in details.FlowActions)
            {
                await SignerClient.SendFlowActionReminderAsync(result.DocumentId, flowAction.Id);
            }
        }
    }
}

File: nodejs/scenarios/notifyFlowParticipantsScenario.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const signer_node_client_1 = require("signer-node-client");
const scenario_1 = require("./scenario");
/**
 * This scenario demonstrates how to notify participants
 * of the flow.
 */
// 1. Get a document Id
(0, scenario_1.CreateDocument)().then((response) => {
    const notification = new signer_node_client_1.NotificationsApi(scenario_1.config);
    const document = new signer_node_client_1.DocumentsApi(scenario_1.config);
    // 2. Get the document details
    document.apiDocumentsIdGet(response.documentId).then((res) => {
        res.data.flowActions.forEach((element) => {
            // 3. Notify each participant individually if necessary
            //    Note: Only participants with pending actions are notified.
            notification
                .apiNotificationsFlowActionReminderPost({
                flowActionId: element.id,
                documentId: response.documentId,
            })
                .then((resp) => console.log("Participante Notified, Status :", resp.status));
        });
    });
});
